charts/retool/values.yaml

```diff
@@ -154,6 +154,9 @@ postgresql:
     database: hammerhead_production
     username: postgres
     password: retool
+    # Uncomment this if setting `username` to `postgres`. If this field is not set, a random password
```

Comment:
why is this by default commented out if the username is by default set to postgres?

Reply:
We can't just add a new default for postgresPassword because that would break anyone who has installed the chart with the username postgres but not have postgresPassword set by changing the password for them.

same thing with the username, we can't just change the default now that it's already been `postgres` for forever. there might be people using these default values and not specifying their own at all, and if we changed it upgrading would break things for them. the best thing we can do here is to make sure the upgrades and future new installs dont break

Reply:
does it make sense to comment out the username field too then? I feel like as is the wording of the comment makes me wonder if i should uncomment it

like basically force the user to end up having to either uncomment (hopefully im following this right)
#username: postgres
#postgresPassword: defaultpassword

or 
#username: your-custom-user
#password: defaultpassword

Reply:
that's still changing the defaults though. a user can have a values.yaml file that only has the license key, jwt secret and encryption key set, and that would give them a working retool instance using the `postgres` user + generated password combo right now. if we comment that out from our defaults, their next upgrade will break

Reply:
oh oops ok

Reply:
can we change the wording to 'IMPORTANT: When setting `username` to `postgres` a random password will be generated unless this field is uncommented'

I think my issue is with the current wording being 'if ur setting something, do x' but it already appears to be set by default